designer/client/outputs/save-per-page-edit.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ValidationErrors } from "./types";
+import { Input } from "@govuk-jsx/input";
+
+type Props = {
+  savePerPageUrl: string;
+  errors: ValidationErrors;
+};
+
+const SavePerPageEdit = ({ savePerPageUrl = "", errors }: Props) => (
```

Comment:
sorry, how does this work. Might be naming thats confusing me

Reply:
Sorry yeah its more like SavePerPageOutputEdit on the designer for creation of the web-hook. Its using the same naming conventions as the other outputs.